### ISSUE TITLE
[Feature Request] Add multiple sources to AutoWallpaper

### ISSUE THREAD

**Issue by joniturmendi** [b-lam/Resplash#154]
Right now, Auto Wallpaper only lets you specify a single source for the pictures. But let's say that I would like my feed of auto wallpapers to consist of a combination of two (or more) independent search terms, or a search term plus a collection, or any combination of the actual available options. That would be a great feature! 

Thank you in advance.

**Comment by joniturmendi** [comment]
@realmenwearpink2  Yes, you can specify as many search terms as you want, but I suppose that the search result would be pictures that satisfy all of the specified terms at once, and not one by one. For example: if I write "car" and "cat" in search terms, I guess I would get pictures with cars AND cats in them, but not pictures of just cars and just cats. My point is that it would be nice to be able to provide independent search terms that would result in a combined feed of pictures.

**Comment by realmenwearpink2** [comment]
I'm pretty sure you can separate search terms with a comma, and it will search those terms independently.
Have you tried that?

**Comment by joniturmendi** [comment]
You are right, that works! I was separating the terms by spaces 😅 anyway, the other feature we mentioned about combining different source types would be great
